pkg/servicelevel/types.go

```diff
@@ -303,6 +319,16 @@ type ServiceLevelResultQuery struct {
 	NRQL NRQL `json:"nrql"`
 }
 
+// User - The `User` object provides general data about the user.
+type User struct {
```

Comment:
This struct is unrelated to the changes on the API. Is this new object expected and harmless?

Reply:
In this case it's probably harmless. Not quite sure why it was generated and added to the `Actor` struct, but sometimes Tutone might generate an extraneous type to satisfy the GraphQL schema.